eZ/Publish/Core/MVC/Symfony/Cache/Http/InstantCachePurger.php

```diff
@@ -47,31 +47,42 @@ public function __construct(
     /**
      * @deprecated as of 6.0. Will be removed in 7.0. Use purgeForContent() instead.
      *
-     * @param mixed $cacheElements
+     * @param array $locationIds
      *
      * @return mixed
      */
-    public function purge($cacheElements)
+    public function purge($locationIds)
     {
-        $this->purgeClient->purge((array)$cacheElements);
+        $this->purgeClient->purge((array)$locationIds);
 
-        return $cacheElements;
+        return $locationIds;
     }
 
     public function purgeAll()
     {
         $this->purgeClient->purgeAll();
     }
 
-    public function purgeForContent($contentId)
+    /**
+     * Purge Content cache using location id's returned from ContentCacheClearEvent.
+     *
+     * @deprecated in 6.5, design flaw on deleted/trashed content, use purge() on cases you are affected by this for now.
```

Comment:
Unclear.. suggestion:
`@deprecated in 6.5 due to a design flaw affecting deleted/trashed content. For now, use purge() in cases where you are affected by this.`